windows/src/Defines.mak

```diff
@@ -146,6 +146,7 @@ TARGET_PATH=Release
 
 WIN32_TARGET_PATH=Win32\$(TARGET_PATH)
 WIN64_TARGET_PATH=Win64\$(TARGET_PATH)
+X64_TARGET_PATH=x64\$(TARGET_PATH)
```

Comment:
I take it WIN64_TARGET is for something different from X64_TARGET?

Reply:
It's a kinda annoying situation where Visual C++ uses names the platform "x64" whereas Delphi names it "win64". I haven't found a clean way to make them mesh better.